docs/web_modules/sfeir-school-theme/README.md

```diff
@@ -1,1046 +0,0 @@
-[![npm version](https://badge.fury.io/js/sfeir-school-theme.svg)](https://badge.fury.io/js/sfeir-school-theme)
```

Comment:
You should not drop the web_modules directory, which is needed for the github pages version